config/crds/v1/crds.kubeflare.io_pagerules.yaml

```diff
@@ -78,10 +78,15 @@ spec:
             - zone
             type: object
           status:
-            description: PageRuleStatus defines the observed state of PageRule
```

Comment:
This is not exactly the value returned by Cloudflare API, it's more of "LastAppliedPriority".

I'm using it because I can't depend on the value of the `priority` field returned from Cloudflare to detect if the `priority` in the `Spec` was changed since Cloudflare re-computes the `priority`. So to detect a change to `priority` in the `Spec`, I compare the value of `priority` in `Spec` with the value of `priority` in `Status`. After a change to `priority` in the `Spec` has been applied to Cloudflare, the `priority` field of the `Status` object is also updated.

So, would `LastAppliedPriority` be a better name or do you think it should be handled differently?